index.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
```suggestion
<?php

```
Agregar una línea después de este tag